js/adapt-youtube.js

```diff
@@ -88,6 +91,29 @@ define(function(require) {
                 }
             }
         },
+        
+        onToggleInlineTranscript: function(event) {
+            if (event) event.preventDefault();
+            var $transcriptBodyContainer = this.$(".youtube-inline-transcript-body-container");
+            var $button = this.$(".youtube-inline-transcript-button");
```

Comment:
You've got [this bug](https://github.com/adaptlearning/adapt_framework/issues/2062) here... you can fix by doing this:
```suggestion
            var $buttonText = this.$(".youtube-inline-transcript-button .transcript-text-container");
```
then changing the instances of `$button` on L105 and L111 to `$buttonText`